Scripts/SL-PlayerOptions.lua

```diff
@@ -402,15 +402,19 @@ local Overrides = {
 			return list
 		end,
 		SaveSelections = function(self, list, pn)
-			local mods = SL[ToEnumShortString(pn)].ActiveModifiers
+			local sl_pn = SL[ToEnumShortString(pn)]
+			local mods = sl_pn.ActiveModifiers
 			if SL.Global.GameMode == "FA+" then
 				 -- always disable in FA+ mode since it's handled engine side.
 				mods.ShowFaPlusWindow = false
 				mods.ShowEXScore   = list[1]
+				sl_pn.EvalPanePrimary = list[1] and 2 or 1
```

Comment:
This should always be 1, because there is no pane 2 in FA+ mode.

Reply:
*facepalm* yeah you're right good catch!